### DISCUSSION TITLE
[RCLONE] Mounted FS Uploaded Filename

### DISCUSSION THREAD

**Discussion by wes1993** [tgdrive/teldrive#184] [Q&A] [answered]
Hello,
I have mounted Telegram using Rclone, if I save file inside the mounted directory Rclone Upload the file with success but there is a way to name the file ulpoaded in the group?

I have seen that in the database and in Teldrive webgui the name is correct but in the group is an alphanumeric name:
![image](https://github.com/divyam234/teldrive/assets/52070589/1f9abe90-c8ec-4ae9-a4a7-03d3cf16906c)

If possible the filename should be the same as the DB / Teldrive Gui.

Best Regards
Stefano

**Comment by puffious** [comment, marked as answer]
https://github.com/divyam234/teldrive#rclone-config-example
set random_chunk_name=false in rclone.conf

**Reply by wes1993** [reply]
Thanks :-D
